docker-compose.yml

```diff
@@ -18,6 +18,9 @@ services:
       UI_URL: http://wecarry.local:5000
       DATABASE_URL: postgres://wecarry:wecarry@db:5432/wecarry?sslmode=disable
       GO_ENV: development
+      AWS_S3_ENDPOINT: http://minio:9000
+      AWS_S3_DISABLE_SSL: "true"
+      AWS_S3_BUCKET: gtis-hca-dev
```

Comment:
Should all of these `gtis-hca-dev` values be changed to substitute something less identifiable?